polonius-engine/src/output/datafrog_opt.rs

```diff
@@ -91,7 +91,6 @@ pub(super) fn compute<Region: Atom, Loan: Atom, Point: Atom>(
 
         let dying_can_reach_live =
             iteration.variable::<((Region, Point, Point), Region)>("dying_can_reach_live");
-        let dying_can_reach_live_r1pq = iteration.variable_indistinct("dying_can_reach_live_r1pq");
```

Comment:
Sorry I should have left a comment before that I had noticed this variable, and when I tried removing it I had the same experience as [yours](https://github.com/rust-lang-nursery/polonius/blob/master/polonius-engine/src/output/datafrog_opt.rs#L89-L90): it was slightly slower on clap here as well. Probably just noise.

Reply:
Yeah. I tried measuring and didn't really notice anything reliable. It's hard to imagine though that it's *better* to have extra variables, right?